---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

Genes in a genome do not play as an orchestra. Selfish elements can play by their own rules and spread in genomes and populations, even when they come at a cost to host fitness. This causes intragenomic conflicts between selfish elements and their host genomes. However, it is difficult to fully assemble these elements with traditional sequencing methods (e.g. Sanger and Illumina sequencing), limiting our understanding of their biology. My research integrates modern genomic analyses (long-read sequencing, ChIP-seq and GWAS) with classic approaches (polytene squashes and FISH) to study how intragenomic conflicts shape genome evolution.
I learned classic genetics, cytological skills and bioinformatics from my Master program training with Dr. Chau-Ti Ting. With these successful experiences, I joined the PhD program at University of Rochester to pursue advanced understanding of chromosome evolution in 2014. I join Dr. Larracuente’s lab and focused on the rapidly evolving parts of genomes that tend to be involved in intragenomic conflicts, including the Y chromosome, the selfish Segregation Distorter chromosomes, and centromeres. I applied long-read sequencing technologies as well as classical cytological methods to generate accurate assemblies, and detailed the organization of these previously unassembled regions. Aside from providing an important resource to study these regions, my dissertation will also reveal that the rapid genomic rearrangements are strongly associated with the evolution of selfish genetic elements.

Y chromosomes are usually gene-poor and enriched in repetitive sequences, yet they can affect many phenotypes and contribute to hybrid incompatibilities between species. My research sheds light on the evolutionary dynamics and mutation patterns of Y chromosomes. Using comparative genomics of D. melanogaster and its three related species in the simulans clade, I revealed different mutation patterns between the Y chromosome and the rest of the genome. Y chromosomes in Drosophila have high duplication and gene conversion rates compared to other chromosomes. The specific mutation patterns help Y chromosomes recruit genes from other chromosomes. I discovered one gene family that moved to the Y chromosome and duplicated to be in many copies on the Y chromosome of simulans clade species. In D. melanogaster, SR Protein Kinase (SRPK) is on an autosome and is essential for both oogenesis and spermatogenesis. After duplicating to the Y chromosome of the ancestor of the simulans clade species, SPRK amplified its copy number to >40 copies. Interestingly, Y-linked SRPK retained exons from a testis-specific isoform deleted in the autosomal copies, in a striking case of subfunctionalization to keep testis-specific functions in the Y-linked copies. While the Y chromosome has very little nucleotide variation in protein-coding gene regions, it has extensive structural variation between species, including different gene content and gene copy number. D. pseudoobscura offers an independent examination of Y chromosome specific mutation patterns. The ancestral Y is now autosomal and 100-fold smaller in size. I assembled the whole chromosome, including the ancestral Y part, into a single contig. I found that the small ancestral Y part of autosomes is primarily due to the reduced size of intergenic regions. In addition, the ancestral Y part of autosome does not have the distinctive Y-linked mutation patterns found Y chromosome in other species, even though it has the same genes. In conclusion, my work suggests that Drosophila Y chromosome has evolved chromosome-specific mutation patterns that parallel those observed on mammalian Y chromosomes, despite their independent evolutionary origins. The unique patterns of duplication and gene conversion help the Y chromosome acquire and maintain genes.  
Centromeres are chromosomal regions where the kinetochores form and the spindle microtubules attach during cell division. Despite being essential for cell division, both the centromeric DNA and kinetochore proteins evolve rapidly. The role of centromeric DNA in centromere function is not well understood. In collaboration between my graduate advisor’s lab and Dr. Barbara Mellone’s lab at the University of Connecticut, we recently discovered the sequences underlying the centromeres of Drosophila melanogaster. I mapped ChIP-seq reads from the centromeric histone variant, CENP-A, to my assembly to identify the candidate centromeric sequences and analyzed the sequences associated with CENP-A. We validated the candidate centromeres using FISH techniques. We found that one retroelement (G2/Jockey-3) is shared among all five centromeres. Our work showed that centromeric DNA of D. melanogaster, similar to many plants and mammals, is rich in centromere-associated transposable elements. We are continuing this collaboration to study the evolution of centromere organization in D. melanogaster and three sister species in the simulans clade.

Finally, I also study selfish genetic elements and their role in maintaining genetic variation in populations. Classical genetic studies have identified many inversions in Drosophila, and suggest that these inversions are maintained by balancing selection. However, there is little molecular evidence about the mechanism that maintains this variation. I hypothesize that arms races between selfish elements and their suppressors play a role in balancing selection on chromosomal inversions. To test my hypothesis, I used the well-studied autosomal male meiotic drive complex in D. melanogaster: Segregation Distorter (SD). SD chromosomes kill wildtype sperm in heterozygous males during spermatogenesis thereby increasing their transmission. SD chromosomes frequently contain inversions that link the distorter and its enhancers. However, we do not know why SD chromosomes are polymorphic for inversions. I surveyed inversion polymorphism on SD chromosomes in US populations using both PCR and polytene squashes. I identified five inversions on SD chromosomes, including two inversions that have not been previously described. I further characterized one suppressor on the X chromosome with various effects on different SD chromosomes, ranging from complete suppression to no suppression. I am mapping the suppressor locus using a combination of recombination mapping and GWAS in 87 strains from Drosophila Genetic Reference Panel. My results suggest that arms races between SD chromosomes and their suppressors might maintain polymorphic inversions on SD chromosomes in the US.


